### PR TITLE
Replaced curry with curryN to save up from 7B to 17B to every edited …

### PR DESCRIPTION
**Pull request by Beraliv**
…method

Need to verify if it's worth it.
56 files with average 12B equals 672B in total.
Please accept PR if you accept the changes.

### CODE REVIEW

File: lib/divide/index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-var curry = require('../curry')
+var curryN = require('../curryN')
 
-export default curry(function dovide(a, b) {
```
Comment (Kelin2025) [inline lib/divide/index.js]: `dovide` lol

Comment (Beraliv) [reply lib/divide/index.js]: yeah, o and i are quite close)

**Comment by Beraliv** [comment]
@Kelin2025 Forgot to mention that now it's necessary to edit ```add-method.js``` as lines 15 and 17 should be replaced with ```curryN``` as well.

**Comment by Kelin2025** [comment]
@BabkinAleksandr you read my thoughts

**Pull request merged**